### PR TITLE
Push meeting options to v1

### PR DESCRIPTION
**Pull request by frankpeng7**
this reverts #10405

### CODE REVIEW

File: concepts/changelog.md [hunk 1]
```diff
@@ -64,8 +70,8 @@ For a summary of the value of these API changes, as well as recent tools, compon
 
 | **Change type** | **Version**   | **Description**                          |
 | :-------------- | :------------ | :--------------------------------------- |
-| Addition | beta | Added the **role** property to the [meetingParticipantInfo](/graph/api/resources/meetingParticipantInfo?view=graph-rest-beta&preserve-view=true) complex type.|
-| Addition | beta | Added new action to retrieve an [onlineMeeting](/graph/api/resources/onlinemeeting?view=graph-rest-beta&preserve-view=true) by JoinWebUrl.|
```
Comment (Lauragra) [inline concepts/changelog.md:68]: Please keep the &preserve-view-true parameter on the URLs; this is to get around an OPS warning. Please also add it to the links in your Cloud communications table.

File: concepts/changelog.md [hunk 2]
```diff
@@ -19,6 +19,12 @@ For a summary of the value of these API changes, as well as recent tools, compon
 | :-------------- | :------------ | :--------------------------------------- |
 | Addition | beta | Added [printTaskDefinition](/graph/api/resources/printtaskdefinition?view=graph-rest-beta&preserve-view=true) to the resources supported for [change notifications](/graph/webhooks).|
 
+### Cloud communications
```
Comment (Lauragra) [inline concepts/changelog.md:22]: Please also add a Changelist item to the Skype.Calling.json file in the changelog folder. We're transitioning to a JSON changelog to support the new filterable changelog experience.

Comment (frankpeng7) [reply concepts/changelog.md:22]: ok but any instruction on how to add an item there? (Where does "id" come from? What's possible values for "ApiChange"?)

Comment (Lauragra) [reply concepts/changelog.md:22]: @frankpeng7 , for the ID, you can just use an online GUID generator. We don't know for sure what all the attributes are for. ApiChange should be the name of the API(s). We're trying to get clarification from the engineering team who designed the JSON. You can also reference the Fields section in this doc: https://microsoft-my.sharepoint-df.com/:w:/r/personal/abmohan_microsoft_com/_layouts/15/guestaccess.aspx?CID=5F43DC5E-724F-43BB-88DA-4F561485866B&wdLOR=cFFC4E62E-46A5-499F-AB5B-17B213BE2EA8&share=EVyksKQxYohFk4yTUfFPJWABqGu22tWR1Xp7YrSKHi3d-g. I can take a look and adjust values if needed.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**